### ISSUE TITLE
Check for and fix Safari problems

### ISSUE THREAD

**Issue by owi92** [elan-ev/tobira#724]
Safari works a little different in a few places. We should try to identify these and fix them.

There are at least two issues that get apparent when using tab-navigation in Tobira:

1. Using a border radius for focus outlines doesn't work. I guess we'll either need to use box-shadow instead or be okay with the focus outline not having rounded corners in Safari.

2. Certain elements simply get skipped and aren't focusable at all. That applies at least for the following elements:
- Logo inside the header and on the login page
- `back` button on the login page
- Login button on both desktop and mobile
- "About Tobira" and "Graphi/QL" buttons inside the footer
- "My content" and "Upload video" items inside the user menu
- Both navigation menus (mobile: burger menu / desktop:left sidebox)
- Links inside the "welcome" text
- Videos in series blocks
- Breadcrumb items
- "Go to page" and "Add sub-page" buttons in "Page settings"
- "View page" button in "Edit page contents"
- Video player on video pages
- "Manage" button on video pages
- ...

**Comment by owi92** [comment]
> Certain elements simply get skipped and aren't focusable at all.

Well turns out this is a safari "feature". Users need to specifically enable this in their browser preferences. The reasoning behind that is beyond me. But... once again I propose closing this without a "fix". I just hope people know what they're getting into when choosing safari as their default browser.

<img width="769" alt="Bildschirmfoto 2023-11-13 um 12 25 27" src="https://github.com/elan-ev/tobira/assets/94838646/6e6b2e9c-0b6d-4f52-9da6-904f4cd9df2f">

**Comment by LukasKalbertodt** [comment]
The heck? Yeah that seems weird. I trust your assessment here, I don't have a desktop Safari to test anyway. If still unsure, you can always summon Julian to express his opinion as well.

**Comment by owi92** [comment]
Sure. @JulianKniephoff, _thou art summoned here to expresseth thy opinion_!

**Comment by JulianKniephoff** [comment]
Looks pretty intentional to me on the part of Safari, so I wouldn't tamper with it. 🤷‍♀️

**Comment by owi92** [comment]
Closed following the above discussion.
